src/api/status/public/index.html

```diff
@@ -1,1235 +0,0 @@
-<!--
-=========================================================
-* Material Dashboard 2 - v3.0.0
-=========================================================
-
-* Product Page: https://www.creative-tim.com/product/material-dashboard
-* Copyright 2021 Creative Tim (https://www.creative-tim.com)
-* Licensed under MIT (https://www.creative-tim.com/license)
-* Coded by Creative Tim
-
-=========================================================
-
-* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
--->
```

Comment:
I just notice there is a MIT license, where in the code I should put this comment?

Reply:
Copy his entire LICENSE file (if not already there) into `src/api/status/public/LICENSE`